src/NServiceBus.Unity/NServiceBus.Unity.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Unity.Container" Version="[5.8.6, 6.0.0)" />
+    <PackageReference Include="Unity.Container" Version="[5.8.6, 5.9.0)" />
```

Comment:
Just an FYI, but this won't prevent someone from trying to use the prerelease versions of 5.9.0 that are nuget.org.

Reply:
is that really a scenario we need to prevent? are you suggesting to use `5.9.0-` instead?

Reply:
Replied on the other PR, but no I don't really think there's anything we can do. Just seeing this upper range being applied to protect us against breaks just reminded me of the bad behavior.